internal/pkg/cli/file/hidden.go

```diff
@@ -0,0 +1,13 @@
+//go:build !windows
+
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package file
+
+import "path/filepath"
+
+// IsHiddenFile returns true if the file is hidden on non-windows.
```

Comment:
`filepath.Base("")` returns `"."`, and we would say an "" is a hidden file. I think we should either make it clear that `""` doesn't work, or just return `false, errors.New()` if the filename is empty.
```suggestion
// IsHiddenFile returns true if the file is hidden on non-windows. The filename must be non-empty.
```